crates/python/src/compiler/quilc.rs

```diff
@@ -103,12 +100,60 @@ impl PyTargetDevice {
     }
 }
 
-wrap_error!(RustRpcqError(rpcq::Error));
-py_wrap_error!(quilc, RustRpcqError, RpcqError, PyRuntimeError);
-
-#[derive(Clone)]
+#[derive(Clone, Debug)]
 pub enum QuilcClient {
```

Comment:
For reference, this would be another enum that would want `#[non_exhaustive]`

Reply:
OTOH, we don't intend for this to be used in _Rust_ code outside of this crate, so maybe not.

Reply:
`#[non_exhaustive]` is new to me -- thanks.

Reply:
But yeah, probably not necessary here.